requirements.txt

```diff
@@ -6,4 +6,4 @@ Flask
 
```

Comment:
It may be useful to introduce pinned/versioned dependencies in requirements.txt to ensure that wgkex still works even if some dependencies have received incompatible updates

Reply:
This PR is already pretty big, I'd prefer not to inflate it even more with unrelated changes.
Do you want to reopen one of your PRs for this? We can try to tackle it together there. But maybe #89 fixes the CI problems you've encountered already.